### PR TITLE
Seo 173713 wpf image alt redirect 12

### PR DESCRIPTION
**Pull request by amyprudencekingstone**
Hi @MallikaRK ,
Worked on wpf image alt redirect, kindly review.

Regards,
Amy Kingstone.

### CODE REVIEW

File: wpf/Barcode/Overview.md [hunk 1]
```diff
@@ -11,21 +11,21 @@ documentation: ug
 
 The Barcode control helps rendering bar codes in desktop (WPF) application. The control can be merged with into any desktop application and easy to encode text using the supported symbol types. The basic structure of a bar code consists of a leading and trailing quiet zone, a start pattern, one or more data characters, optionally one or two check characters, and a stop pattern. 
 
-![WPF One Dimensional Barcode](Overview_images/wpf-one-dimensional-barcode.png)
+![One dimensional in Barcode WPF](overview_images/wpf-barcode-one-dimensional.png)
```
Comment (MallikaRK) [inline wpf/Barcode/Overview.md:14]: revert your ALT name changes, exiting alt text seems fine.

File: wpf/Barcode/Overview.md [hunk 2]
```diff
@@ -11,21 +11,21 @@ documentation: ug
...
 
 Barcode control rendering 1D bar code
 {:.caption}
 
 
-![WPF Two Dimensional Barcode](Overview_images/wpf-two-dimensional-barcode.png)
+![Two dimensional in Barcode WPF](overview_images/wpf-barcode-two-dimensional.png)
```
Comment (MallikaRK) [inline wpf/Barcode/Overview.md:20]: revert your ALT name changes, exiting alt text seems fine.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/Barcode/Overview.md (hunk 1 above)
Comment (amyprudencekingstone) [reply wpf/Barcode/Overview.md]: made changes

File: wpf/Barcode/Overview.md (hunk 2 above)
Comment (amyprudencekingstone) [reply wpf/Barcode/Overview.md]: made changes

**Pull request closed without merging**